pkg/helmclient/rollback.go

```diff
@@ -22,6 +22,8 @@ func (c *Client) Rollback(ctx context.Context, namespace, releaseName string, re
 		return microerror.Mask(err)
 	}
 
+	eventCounter.WithLabelValues(eventName).Inc()
```

Comment:
In this case, if we have a problem during `c.rollback` function call, the counter value would not increase. 
What do you think to call it with `defer` above the rollback line?

Reply:
Good point. I used an anonymous func so the timer includes counting the rollback. PTAL

Reply:
Thanks, maybe not only rollback but install, update also need the same changes, too. :)

Reply:
There are already operatorkit metrics for errors. But OK lets use the same approach for each counter. Done PTAL